### DISCUSSION TITLE
Feature Request: Favicon transparency

### DISCUSSION THREAD

**Discussion by sonjoonho** [omnivore-app/omnivore#2283] [Ideas]
A small and possibly selfish feature request: the web app and extension icons have transparent background which makes them hard to see with my Chrome theme. It would be nice to have a solid background, or perhaps a different icon colour.

![image](https://github.com/omnivore-app/omnivore/assets/20669173/5d66a445-5f1f-498f-93b7-e0b750f7af41)

**Comment by jacksonh** [comment]
hah, yeah that sounds reasonable!

**Comment by sonjoonho** [comment]
Looks like this was addressed (: not sure how it would look on a light theme though.

![image](https://github.com/omnivore-app/omnivore/assets/20669173/4f7d2382-e4a3-435b-9de4-a9ca9a1af809)
